pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -1421,6 +1421,9 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
                   console.log(`Disabling flannel and network policy`);
                   k3sConf.ADDITIONAL_ARGS += ' --flannel-backend=none --disable-network-policy';
                 }
+                if (config.application.debug) {
```

Comment:
```suggestion
                if (this.debug) {
```
This doesn't seem to be available in lima's context, though.

Reply:
I just copied this from the surrounding code, and was annoyed that one version uses `cfg` and the other `config`. This code will all be removed in the big refactoring.